lib/modules/showImages.js

```diff
@@ -908,6 +908,16 @@ function shouldLinkBeDeferred(element) {
 const inText = element => !!element.closest('.md, .search-result-footer');
 
 async function checkElementForMedia(element) {
+	// Prevents relatively addressed anchor link urls from refreshing the page,
+	// while also opening all #res:settings urls on the current page
+	const hash = (/^#.*|#res:settings.*/gi).exec(element.getAttribute('href'));
```

Comment:
It would be clearer to test this with something like `linkElement.hash.startsWith('#res:settings')`

Reply:
The regex is doing a couple of addition checks here, though.  

The expression is basically  
`if ((element.getAttribute('href').startsWith("#") || element.hash.toLowerCase().startsWith("#res:settings"))`

**Note:** _For some reason, the `element.href` always returns a fully qualified link, even when the actual `href` attribute is only "#anchorLink", so I needed to compare it to the ".getAttribute('href')" to know if it's a fully qualified link or just an anchor link by itself. The fully qualified links are not meant to be handled unless it's a #res:settings link._ 

So if you guys just prefer less regex's, I can use that, but if you just assumed it was doing the equivalent to `linkElement.hash.startsWith('#res:settings')`, perhaps you might prefer the regex since it's doing a few things checks at once.